build/package/Microsoft.DotNet.Cli.Installer.targets

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    
+    <Target Name="GenerateInstallers" 
+          DependsOnTargets="Init;Layout;GeneratePkgs;GenerateDebs;GenerateMsis" />
```

Comment:
It just seems odd to have a file with a single method in it and no vision for adding any more methods to it.  I agree splitting things up into cohesive files makes sense, but there is a balance between everything being in its own file, and everything in one big file.

One big file == hard to read.  

But everything in its own file is also hard to read because you are constantly switching between many files trying to get the "big picture" in your head.